nntrainer/models/neuralnet.cpp

```diff
@@ -1152,4 +1155,12 @@ void NeuralNetwork::forEachLayer(
     fn(*ln, std::forward<RunLayerContext &>(ln->getRunContext()), user_data);
   };
 }
+
+void NeuralNetwork::serialize(const std::string file_path) {
```

Comment:
How about something like
``` c++
void NeuralNetwork::exports(nntrainer::ExportMethods method, const std::string file_path)
```

and call the tflite serialize inside. Cause we will support more than just tflite serialize.